draft-ietf-tls-external-psk-importer.md

```diff
@@ -138,14 +178,70 @@ EPSKs may be imported for early data use if they are bound to protocol settings
 otherwise be required for early data with normal (ticket-based PSK) resumption. Minimally, that means ALPN,
 QUIC transport settings, etc., must be provisioned alongside these EPSKs.
 
+##Client ID {#client-id-description}
+The `client_id` is an optional field that is required to be unique for each actor that knows the EPSK.
+This is to prevent Selfie-style reflections.
+If two actors use the same `client_id` this defence may not be effective. 
+This is only necessary in scenarios where more than two actors use the same key, including the case where a single agent will complete PSK handshakes as both the client and the server using the same key.
+
+The Selfie attack {{Selfie}} abuses an underlying assumption of TLS, that only one client and one server know a PSK.
+A pair of agents that both act as both client *and* server, and use the same PSK in both roles violate this assumption.
+In this case, there are two clients and two servers that know the PSK, since each endpoint acts as both client and server.
+
+Because both clients have identical configurations, an agent acting as a server can't distinguish between its peer acting as a client, and itself acting as a client.
```

Comment:
```suggestion
Because both clients have identical configurations, an agent acting as a server cannot distinguish between its peer acting as a client, and itself acting as a client.
```